File: python/Kode program/RoomInfo.py
class RoomInfo:
    def __init__(self):
        self.roomName = ""
        self.floorLevel = 0

    def setRoomName(self, rn):
        self.roomName = rn

    def getRoomName(self):
        return self.roomName

    def setFloorLevel(self, fl):
        self.floorLevel = fl

    def getFloorLevel(self):
        return self.floorLevel

    def printRoom(self):
        print("=== Room Info ===")
        print(f"Room Name: {self.roomName}")
        print(f"Floor Level: {self.floorLevel}")

File: java/Kode program/RoomInfo.java
public class RoomInfo {
    private String roomName;
    private int floorLevel;

    public void setRoomName(String rn) { roomName = rn; }
    public String getRoomName() { return roomName; }

    public void setFloorLevel(int fl) { floorLevel = fl; }
    public int getFloorLevel() { return floorLevel; }

    public void printRoom() {
        System.out.println("=== Room Info ===");
        System.out.println("Room Name: " + roomName);
        System.out.println("Floor Level: " + floorLevel);
    }
}
